data/tests/sign.sh

```diff
@@ -0,0 +1,28 @@
+#/bin/bash
```

Comment:
Yeah, it's just used for build time (when test suite is run).  It wouldn't lead to any runtime dependencies.
The package you'll need for debian is `gnutls-bin`